src/gameplayv2/resistance/entities/Entity.ts

```diff
@@ -0,0 +1,37 @@
+class Entity{
```

Comment:
Let's call this `class Role` instead and have `entity` as a comment. We'll want to use more domain specific language (DSL) if possible to keep the code relevant to the business problem.